examples/hello-world/hello-km/jobs/kaplan-meier/app/custom/km_train.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+from km_analysis import kaplan_meier_analysis
+
+# (1) import nvflare client API
+import nvflare.client as flare
+from nvflare.app_common.abstract.fl_model import FLModel, ParamsType
+
+# Client training code
+
+
+def load_data():
+    data = {
+        "site-1": {
+            "duration": [5, 10, 15, 25, 30, 35, 40, 45, 50, 55, 60, 65],
+            "event": [1, 0, 1, 1, 1, 0, 0, 1, 1, 1, 2, 4],
+        },
+        "site-2": {"duration": [10, 25, 30, 40, 50, 60, 70], "event": [1, 1, 0, 1, 0, 3, 4]},
+    }
```

Comment:
KM by lifelines currently do not support "event histogram", the input event is binary - True if the the death was observed, False if the event was lost (right-censored), so numbers > 1 does not make sense here, and event cannot be directly added (https://lifelines.readthedocs.io/en/latest/fitters/univariate/KaplanMeierFitter.html).
Let's replace this toy data with a public dataset (from sksurv.datasets import load_veterans_lung_cancer)

Reply:
I implemented a HE version based on this one (https://github.com/NVIDIA/NVFlare/pull/2259), the data / aggregation part can transfer here.

Reply:
sounds good